quickstart/101-aci-linuxcontainer-public-ip/main.tf

```diff
@@ -0,0 +1,29 @@
+resource "random_pet" "rg_name" {
+  prefix = var.resource_group_name_prefix
+}
+
+resource "azurerm_resource_group" "rg" {
+  name     = random_pet.rg_name.id
+  location = var.resource_group_location
+}
+
+resource "azurerm_container_group" "container" {
+  name                = var.name
```

Comment:
This variable should be more specific about what the name is for.

Reply:
According to the Bicep version of this article, "In this quickstart, you use a Bicep file to deploy an isolated Docker container and make its web application available with a public IP address." I'm not sure what name would be more appropriate here.